src/ui/about.js

```diff
@@ -100,6 +103,32 @@ function About(props) {
           </React.Fragment>
         )}
 
+        <Styled.h2>Credits</Styled.h2>
+        <Styled.p>
+          Thanks to the following people and institutions for contributing to this project:
+        </Styled.p>
+        <ul>
+          <li>
+            <Styled.a href="https://github.com/slampunk">Duncan Smith</Styled.a> for starting this
+            project
+          </li>
+          <li>
+            <Styled.a href="https://github.com/cilt-uct">University of Cape Town (CILT)</Styled.a>
+            {' '}for letting Duncan start the project
+          </li>
+          <li>
+            <Styled.a href="https://ethz.ch">ETH Zürich</Styled.a> for financial support and
+            testing
+          </li>
+          <li>
+            <Styled.a href="https://github.com/elan-ev">ELAN e.V.</Styled.a> for the final
+            development
```

Comment:
Not to be nit-picky, but "final development" sells us short I think. Basically every aspect of the app was rewritten by us, many things improved. It's absolutely correct to mention Duncan for the idea and the initial implementation, but yeah, we didn't just do some polishing.

Maybe just replace "final" with "continued"? Or "for the continued development and maintenance"?